benchmarks/.gitignore

```diff
@@ -0,0 +1 @@
+/benchmarks_output.json
```

Comment:
should we just put a line in the original `.gitignore` file, like `*.json` (just ignore all `json` files)
I think it would be better to gather all the `.gitignore files together`

Reply:
Ok fine, although it's too restrictive maybe. But for the moment we are not using the json files, so it's fine.